packages/example-react/package.json

```diff
@@ -5,7 +5,7 @@
     "description": "",
     "main": "index.js",
     "scripts": {
-        "storybook": "start-storybook",
+        "storybook": "start-storybook --no-manager-cache",
```

Comment:
https://github.com/storybookjs/storybook/pull/16653 should fix the issue

Reply:
I've tested it locally and it still seems to need --no-manager-cache :/

Reply:
@joshwooding can you send a repro?

Reply:
@tmeasday I'm still seeing the same issue using the repo in https://github.com/storybookjs/storybook/issues/16647

Reply:
Can confirm this is now fixed 🎉